VERSIONING.md

```diff
@@ -25,10 +25,25 @@ Follow these steps when an update is ready to be pushed to Docker Hub:
 
 ## Build Example
 
+Make sure to authenticate with GitHub Container Registry first, see below.
+
 ```
 docker build . -t debian-mail-overlay
-docker tag debian-mail-overlay mailserver2/debian-mail-overlay:0.0.0
-docker push mailserver2/debian-mail-overlay:0.0.0
-docker tag debian-mail-overlay mailserver2/debian-mail-overlay:latest
-docker push mailserver2/debian-mail-overlay:latest
+docker tag debian-mail-overlay ghcr.io/mailserver2/debian-mail-overlay:0.0.0
+docker push ghcr.io/mailserver2/debian-mail-overlay:0.0.0
+docker tag debian-mail-overlay ghcr.io/mailserver2/debian-mail-overlay:latest
+docker push ghcr.io/mailserver2/debian-mail-overlay:latest
+```
+
+## Authenticating with GitHub Container Registry
+
+Create a [personal access token](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-with-a-personal-access-token-classic) by going [here](https://github.com/settings/tokens/new?scopes=write:packages,delete:packages).
```

Comment:
NIT: do we want to add:
`Make sure you set expiration date for your personal token`
?

Reply:
I assumed this was mostly for the maintainers and the maintainers should ideally know good security practices. Right now it's just the three of us and I hope we are in agreement about that. If you think it's important enough I do not mind adding it, but I did consider it and decided that I should not be teaching you guys what your security practices should be ;)